### ISSUE TITLE
%ld used instead of %lld in chunk.c

### ISSUE THREAD

**Issue by patlefort** [AcademySoftwareFoundation/openexr#1255]
Minor problem:

```bash
src/lib/OpenEXRCore/chunk.c:350:86: warning: format ‘%ld’ expects argument of type ‘long int’, but argument 4 has type ‘int64_t’ {aka ‘long long int’} [-Wformat=]
  350 |                 "Invalid chunk size reconstructing chunk table: found out of range %ld",
      |                                                                                    ~~^
      |                                                                                      |
      |                                                                                      long int
      |                                                                                    %lld
  351 |                 leaderdata->deep_data[1]);
      |                 ~~~~~~~~~~~~~~~~~~~~~~~~
      |                                      |
      |                                      int64_t {aka long long int}
```

**Comment by kmilos** [comment]
This is platform dependent, so for portability it is best to use the [inttypes.h print format macros](https://en.cppreference.com/w/c/types/integer#Format_macro_constants) (`PRId64` in this case).

**Comment by cary-ilm** [comment]
Should be resolved by #1287
